java/src/main/native/cmake/Modules/ConfigureNvcomp.cmake

```diff
@@ -28,6 +28,12 @@ elseif(CMAKE_CXX11_ABI)
     list(APPEND NVCOMP_CMAKE_ARGS " -DCMAKE_CXX_FLAGS=-D_GLIBCXX_USE_CXX11_ABI=1")
 endif(NOT CMAKE_CXX11_ABI)
 
+if(PER_THREAD_DEFAULT_STREAM)
+    message(STATUS "NVCOMP: Enabling per-thread default stream")
+    list(APPEND NVCOMP_CMAKE_ARGS " -DCMAKE_C_FLAGS=-DCUDA_API_PER_THREAD_DEFAULT_STREAM")
+    list(APPEND NVCOMP_CMAKE_ARGS " -DCMAKE_CXX_FLAGS=-DCUDA_API_PER_THREAD_DEFAULT_STREAM")
```

Comment:
Will this approach work if nvcomp's CMakeLists.txt sets either of these variables?  It does so conditionally as of the 1.1 release.  I'm worried about this working today but then silently not working if nvcomp updates their CMakeLists.txt in the future to unconditionally set these variables and clobber any setting we're trying to specify on the command-line.